Libplanet/Net/NetMQTransport.cs

```diff
@@ -1011,6 +975,16 @@ private void ValidateSender(Peer peer)
                 TaskScheduler.Default
             );
 
+        private List<Task> BindingMultipleProxies(
+            CancellationToken cancellationToken,
```

Comment:
Usually `CancellationToken` go the last in parameters, and has a `default` value as well.  Could you adjust this?